snippets/input-numbers.js

```diff
@@ -0,0 +1,20 @@
+/* This js snippet is used to add a phone number, social secruity number, or others*/
```

Comment:
This comment includes site-specific descriptions (like `phone number`), so let's make it more generic.
i.e. This snippet can be used mostly when just setting input value is not enough but firing an explicit input event is required to actually change the value behind the framework used.

In general, we write the following items as comments in our snippets:
- In what situation user may want to use this snippet
- All variables to be specified (Always bring these to the top, so the users won't have to make further changes)

Example:
https://github.com/autifyhq/autify-javascript-snippets/blob/master/snippets/assert-checked.js

Could you update the comments this way?  Thanks!